README.md

```diff
@@ -21,7 +21,7 @@ builds of the same package.  With Spack, you can build your software
 *all* the ways you want to.
 
 See the
-[Feature Overview](http://spack.readthedocs.io/en/latest/features.html)
+[Feature Overview](https://spack.readthedocs.io/en/latest/features.html)
```

Comment:
This is worthwhile but I don't think this PR is the place to make these sorts of changes.

Reply:
Yes, this will go away in my new PR : https://github.com/spack/spack/pull/19345